core/src/main/java/io/keploy/regression/keploy/Filter.java

```diff
@@ -7,11 +7,13 @@
 @Setter
 @NoArgsConstructor
 public class Filter {
-    private String UrlRegex;
+    private String acceptUrlRegex;
     private String[] HeaderRegex;
```

Comment:
Instead of only HeadersRegex, it should be `rejectHeaderRegex` & `acceptHeaderRegex`.